src/purchase-bill/purchase-bill.controller.ts

```diff
@@ -16,8 +16,8 @@ export class PurchaseBillController {
   })
   @Post()
   create(@Body() createPurchaseBillDto: CreatePurchaseBillDto) {
-    const id = createPurchaseBillDto.id;
-    return this.purchaseBillService.create(createPurchaseBillDto, id);
+    const vendor_id = createPurchaseBillDto.vendor_id;
+    return this.purchaseBillService.create(createPurchaseBillDto, vendor_id);
```

Comment:
if we have vendorId in the dto, why are we separating the vendorID here. we have the value in the dto and we can simply fetch that value from the dto in the repository

Reply:
corrected that in the next commit